Remotion/Web/Core/Res/HTML/DropDownMenu.js

```diff
@@ -177,6 +178,8 @@ function DropDownMenu_OnClick(context, menuID, getSelectionCount, evt)
 {
   ArgumentUtility.CheckNotNullAndTypeIsJQuery('context', context);
 
+  $('#' + menuID).attr(_dropDownMenu_button_timestampAttributeKey, new Date().getTime());
```

Comment:
data-attribute, please. I think there's a special method for data-attributes  in jquery.